Exiled.CustomModules/API/Features/CustomRoles/CustomTeam.cs

```diff
@@ -35,6 +35,9 @@ namespace Exiled.CustomModules.API.Features.CustomRoles
     /// </remarks>
     public abstract class CustomTeam : CustomModule
     {
+        private const string VanilaConfigMtfMinTime = "minimum_MTF_time_to_spawn";
+        private const string VanilaConfigMtfMaxTime = "maximum_MTF_time_to_spawn";
+
```

Comment:
Again, I don't see how moving immutable strings to a constant variable is any different to just hard-referencing the exact same string.

Reply:
https://github.com/Exiled-Team/EXILED/pull/2764#discussion_r1712663457

Reply:
https://github.com/Exiled-Team/EXILED/pull/2764#discussion_r1712665478